core/src/doc/mod.rs

```diff
@@ -34,6 +34,7 @@ mod lives; // Processes any live queries relevant for this document
 mod merge; // Merges any field changes for an INSERT statement
 mod pluck; // Pulls the projected expressions from the document
 mod purge; // Deletes this document, and any edges or indexes
+mod relation;
```

Comment:
Can we add a mod comment here to keep it consistent?

Reply:
I've added a comment, is it ok?